### PR TITLE
integrate op weather coz bootiful [2/2]

### PR DESCRIPTION
**Pull request by 4PERTURE**
I don't like git

### CODE REVIEW

File: config/common.mk [hunk 1]
```diff
@@ -165,6 +165,7 @@ PRODUCT_PACKAGES += \
     PinkPipe \
     WallpaperPicker \
     WeatherProvider
```
Comment (divadsn) [inline config/common.mk]: Missing `\`

Comment (4PERTURE) [reply config/common.mk]: In weatherprovider? Not my doing

**Comment by 4PERTURE** [comment]
I realize now that I am stupid. I will fix this

**Comment by 4PERTURE** [comment]
Fixed

**Comment by edi184** [comment]
Idk if we need that

**Pull request merged**